boards/phytec/phyboard_pollux/doc/index.rst

```diff
@@ -72,6 +72,10 @@ features:
 | GPIO      | on-chip    | GPIO output                        |
 |           |            | GPIO input                         |
 +-----------+------------+------------------------------------+
+| I2C       | on-chip    | ii2c                               |
```

Comment:
```suggestion
| I2C       | on-chip    | i2c                               |
```

Reply:
The nxp-ii2c driver is used, I wanted to state that here. Should I use i2c or ii2c? Or do you think it would be better to state that i2c-master / slave are supported?

Reply:
ah okay. Raw headers says `Driver/Component` but content is a little bit mixed IMO. Something that could be improvied in the future.

`ii2c` is fine. Thought it's a typo.